src/cpager/cpager.c

```diff
@@ -1,35 +1,80 @@
+#include <criu.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <fcntl.h>
 #include <Python.h>
 
+#define RAISE_PYTHON_ERROR_ON_FAIL(arg) if (arg < 0) { PyErr_Format(PyExc_ValueError, "%s (%d)", strerror(errno), arg); return NULL; }
```

Comment:
Your should do this multiple lines  -- it's not readble like this..
use \ at the end of line to do multiple line macro